cli/src/test/java/bisq/cli/opt/OptionParsersTest.java

```diff
@@ -0,0 +1,160 @@
+    }
+
+    @Test
+    public void testCancelOfferWithMissingOfferIdValueShouldThrowException() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                canceloffer.name(),
+                "--" + OPT_OFFER_ID // missing equals sign & opt value
+        };
+        Throwable exception = assertThrows(RuntimeException.class, () ->
+                new CancelOfferOptionParser(args).parse());
+        assertEquals("Option offer-id requires an argument", exception.getMessage());
+    }
+
+    @Test
+    public void testValidCancelOfferOpts() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                canceloffer.name(),
+                "--" + OPT_OFFER_ID + "=" + "ABC-OFFER-ID"
+        };
+        new CancelOfferOptionParser(args).parse();
+    }
+
+    // CreateOffer opt parsing tests
+
+    @Test
+    public void testCreateOfferOptParserWithMissingPaymentAccountIdOptShouldThrowException() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                createoffer.name()
+        };
+        Throwable exception = assertThrows(RuntimeException.class, () ->
+                new CreateOfferOptionParser(args).parse());
+        assertEquals("no payment account id specified", exception.getMessage());
+    }
+
+    @Test
+    public void testCreateOfferOptParserWithMissingDirectionOptShouldThrowException() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                createoffer.name(),
+                "--" + OPT_PAYMENT_ACCOUNT + "=" + "abc-payment-acct-id-123"
+        };
+        Throwable exception = assertThrows(RuntimeException.class, () ->
+                new CreateOfferOptionParser(args).parse());
+        assertEquals("no direction (buy|sell) specified", exception.getMessage());
+    }
+
+    @Test
+    public void testCreateOfferOptParserWithMissingDirectionOptValueShouldThrowException() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                createoffer.name(),
+                "--" + OPT_PAYMENT_ACCOUNT + "=" + "abc-payment-acct-id-123",
+                "--" + OPT_DIRECTION + "=" + ""
+        };
+        Throwable exception = assertThrows(RuntimeException.class, () ->
+                new CreateOfferOptionParser(args).parse());
+        assertEquals("no direction (buy|sell) specified", exception.getMessage());
+    }
+
+    @Test
+    public void testValidCreateOfferOpts() {
+        String[] args = new String[]{
+                PASSWORD_OPT,
+                createoffer.name(),
+                "--" + OPT_PAYMENT_ACCOUNT + "=" + "abc-payment-acct-id-123",
+                "--" + OPT_DIRECTION + "=" + "BUY",
+                "--" + OPT_CURRENCY_CODE + "=" + "EUR",
+                "--" + OPT_AMOUNT + "=" + "0.125",
+                "--" + OPT_MKT_PRICE_MARGIN + "=" + "0.0",
+                "--" + OPT_SECURITY_DEPOSIT + "=" + "25.0"
+        };
```

Comment:
Is there an assert missing here?

Reply:
Added in commit 70da6d19fa694c5ae11212645fdfa4ea47529854.

Reply:
In next PR ([5284](https://github.com/bisq-network/bisq/pull/5284)) more asserts were added to check valid, parsed opt values in commit 389abf4.

```
	assertEquals("abc-payment-acct-id-123", parser.getPaymentAccountId());
	assertEquals("BUY", parser.getDirection());
	assertEquals("EUR", parser.getCurrencyCode());
	assertEquals("0.125", parser.getAmount());
	assertEquals("0.0", parser.getMktPriceMargin());
	assertEquals("25.0", parser.getSecurityDeposit());
```